build-system/tasks/process-github-issues.js

```diff
@@ -119,44 +135,91 @@ function updateGitHubIssues() {
           let milestoneTitle;
           let milestoneState;
           let hasPriority = false;
+          let hasCategory = false;
           let issueNewMilestone = MILESTONE_PENDING_TRIAGE;
-
+          const assignee = issue['assignee'];
+          let assigneeName = '';
+          const issueLastUpdate = issue.updated_at;
+          let biweeklyUpdate = true;
+          let quartelyUpdate = true;
+          if (getLastUpdate(issueLastUpdate) > QUARTERLY_DAYS) {
+            quartelyUpdate = false;
+            biweeklyUpdate = false;
+          } else if (getLastUpdate(issueLastUpdate) > BIWEEKLY_DAYS) {
+            biweeklyUpdate = false;
+          }
+          // Get the assignee
+          if (assignee) {
+            assigneeName = '@' + assignee['login'];
+          }
           // Get the title and state of the milestone
           if (milestone) {
-            milestoneTitle = milestone.title;
-            milestoneState = milestone.state;
+            milestoneTitle = milestone['title'];
+            milestoneState = milestone['state'];
+            issueNewMilestone = milestone['number'];
           }
-          // Get the labels we want to check
-          labels.forEach(function(label) {
-            if (label) {
-              // Check if the issues has type
-              if (label.name.startsWith('Type') ||
-             label.name.startsWith('Related')) {
-                issueType = label.name;
-              }
-              // Check if the issues has Priority
-              if (label.name.startsWith('P0') ||
-                  label.name.startsWith('P1') ||
-                  label.name.startsWith('P2') ||
-                  label.name.startsWith('P3')) {
-                hasPriority = true;
-              }
-            }
-          });
+          // promise starts
           promise = promise.then(function() {
             util.log('Update ' + issue.number);
             const updates = [];
-            // Milestone task: move issue from closed milestone
+            // Get the labels we want to check
+            labels.forEach(function(label) {
+              if (label) {
+              // Check if the issues has type
+                if (label.name.startsWith('Type') ||
+               label.name.startsWith('Related')) {
+                  issueType = label.name;
+                }
+                // Check if the issues has Priority
+                if (label.name.startsWith('P0') ||
+                    label.name.startsWith('P1') ||
+                    label.name.startsWith('P2') ||
+                    label.name.startsWith('P3')) {
+                  hasPriority = true;
+                  if (label.name.startsWith('P0')
+                      || label.name.startsWith('P1')) {
```

Comment:
nit: For consistency, move `||` to previous line, similar to the above `if` block.